content/en/docs/ops/configuration/extensibility/wasm-module-distribution/index.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Distributing WebAssembly Modules
+description: Describes how to make remote WebAssembly modules available in the mesh.
+weight: 10
+aliases:
+  - /help/ops/extensibility/distribute-remote-wasm-module
+  - /docs/ops/extensibility/distribute-remote-wasm-module
+keywords: [extensibility,Wasm,WebAssembly]
+owner: istio/wg-policies-and-telemetry-maintainers
+test: n/a
```

Comment:
If this is not an experimental feature, it is applicable., so there should be a test added asap.

```suggestion
test: no
```